docs/reference/commandline/network_create.md

```diff
@@ -122,6 +122,24 @@ docker network create -d overlay
 ```
 Be sure that your subnetworks do not overlap. If they do, the network create fails and Engine returns an error.
 
+# Specifying specific options for the bridge driver
```

Comment:
Perhaps just

```markdown
### Bridge driver options
```

To be consistent with https://github.com/docker/docker/blob/master/docs/admin/logging/overview.md